.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+module.exports = {
+  env: {
+    es6: true,
+    node: true
+  },
+  extends: ['eslint:recommended'],
+  plugins: ['prettier'],
+  rules: {
+    curly: 'error',
+    'no-console': 'off',
+    'no-else-return': 'error',
+    'no-inner-declarations': 'off',
+    'no-var': 'error',
+    'prefer-arrow-callback': 'error',
+    'prefer-const': 'error',
+    'prettier/prettier': ['error', { singleQuote: true }],
+    strict: 'error'
+  }
```

Comment:
I added some more rules here https://github.com/halfzebra/create-elm-app/pull/65. Can you add it one again?